### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by kelleg1** [iaincollins/nextjs-starter#69]
Is this code licensed under MIT or WTFPL or similar? We might like to use it in our organization.

**Comment by iaincollins** [comment]
Yes it's licensed under ISC (an MIT style licence, just shorter).

This was specified in the package.json but I have now added an explicit LICENCE.txt file.
